### PR TITLE
finish adding timestamp_ns/ms/sec/tz datatypes

### PR DESCRIPTION
**Pull request by AEsir777**
Added support for timestamp_ns/_ms/_sec/_tz datatypes
- for node.js, c, following duckdb manner, so there is no timezone for timestamp_tz
- for java, Instant does not support timezone
- python supports timezone for arrow and python api. Numpy/pandas current only support datetime_us to have time zone information
- Rust already contains UTC
- Renabled some test_to_arrow test

### CODE REVIEW

File: src/c_api/prepared_statement.cpp [hunk 1]
```diff
@@ -118,6 +118,30 @@ void kuzu_prepared_statement_bind_date(
     kuzu_prepared_statement_bind_cpp_value(prepared_statement, param_name, std::move(value_ptr));
 }
 
+void kuzu_prepared_statement_bind_timestamp_ns(kuzu_prepared_statement* prepared_statement,
```
Comment (acquamarin) [inline src/c_api/prepared_statement.cpp:121]: I will let @mewim review the api related stuff

File: src/common/types/timestamp_t.cpp [hunk 2]
```diff
@@ -333,5 +333,17 @@ int64_t Timestamp::getEpochNanoSeconds(const timestamp_t& timestamp) {
     return result;
 }
 
+int64_t Timestamp::getEpochMicroSeconds(const timestamp_t& timestamp) {
```
Comment (acquamarin) [inline src/common/types/timestamp_t.cpp:336]: You probably don't need this function, as you can get micos by timestamp.value

**Review by acquamarin — APPROVED** [review]
You may also have to add the timestamp support to scanPandas

File: tools/nodejs_api/test/test_data_type.js [hunk 3]
```diff
@@ -235,6 +235,82 @@ describe("TIMESTAMP", function () {
...
+  it("should convert TIMESTAMP_TZ type", async function () {
+    const queryResult = await conn.query(
+        "MATCH (a:movies) WHERE a.length = 126 RETURN a.description;"
+    );
+    const result = await queryResult.getAll();
+    assert.equal(result.length, 1);
+    assert.equal(Object.keys(result[0]).length, 1);
+    assert.isTrue("a.description" in result[0]);
+    assert.isTrue("release_tz" in result[0]["a.description"]);
+    console.log(result[0]);
```
Comment (mewim) [inline tools/nodejs_api/test/test_data_type.js:248]: Let's remove `console.log` from test case

File: tools/nodejs_api/test/test_data_type.js [hunk 4]
```diff
@@ -235,6 +235,82 @@ describe("TIMESTAMP", function () {
...
+  it("should convert TIMESTAMP_MS type", async function () {
+    const queryResult = await conn.query(
+        "MATCH (a:movies) WHERE a.length = 126 RETURN a.description;"
+    );
+    const result = await queryResult.getAll();
+    assert.equal(result.length, 1);
+    assert.equal(Object.keys(result[0]).length, 1);
+    assert.isTrue("a.description" in result[0]);
+    assert.isTrue("release_ms" in result[0]["a.description"]);
+    console.log(result[0]);
```
Comment (mewim) [inline tools/nodejs_api/test/test_data_type.js:286]: Same here

**Pull request merged**